File: dayThree/java/ReverseAnInteger.java
public class ReverseAnInteger{
    public static void main(String[] args){
        int value = 12345;
        System.out.println(getTheReverseOfInteger(value));
    }
    
    public static int getTheReverseOfInteger(int number){
    
        int reverse = 0;
        
        while(number > 0){
            int lastDigit = number % 10;
            reverse = (reverse * 10) + lastDigit;
            number /= 10;
        }
        return reverse;
    }
}

File: dayThree/javaScript/ReverseAnInteger.js


function getTheReverseOfInteger(number){
    
    let reverse = 0;

    while(number > 0){
        let lastDigit = number % 10;
        reverse = (reverse * 10) + lastDigit;
        number = Math.floor(number/10);
    }
    return reverse;
}
    
let value = 12345;
console.log(getTheReverseOfInteger(value));
    
    
